code/modules/clothing/suits/bio.dm

```diff
@@ -18,7 +18,7 @@
 	w_class = WEIGHT_CLASS_BULKY
 	gas_transfer_coefficient = 0.01
 	permeability_coefficient = 0.01
-	clothing_flags = THICKMATERIAL
+	clothing_flags = THICKMATERIAL | BLOCK_ARTIFACT
```

Comment:
Perhaps this should be something like BLOCK_ARTIFACT_EFFECT, block_artifact just immediately indicates that it has something to do with the blocking mechanics rather than the artifact effects.